common/app/model/TopMentionsModel.scala

```diff
@@ -29,5 +29,7 @@ object TopMentionEntity extends Enumeration {
   val WorkOfArt = Value(5, "WORK_OF_ART")
   val Loc = Value(6, "LOC")
 
+  def withNameOpt(s: String): Option[Value] = values.find(_.toString == s)
```

Comment:
worth doing `_.toString.toUpercase` here instead considering it should always be upercased?

Reply:
good question. I was thinking to let the filter entity to be case insensitive so to allow all forms of following:
`org:Aemo`
`Org:Aemo`
`ORG:Aemo`

But do you think it might be better to strict it and only accept when it's uppercase like `ORG:Aemo`?

Reply:
I think the idea of using toUpercase is to make it not be case sensitive and allow any capitalization of `org`, I think its a good idea to keep it that way, I was just suggesting that the toUpercase be moved into withNameOpt instead of it being the callers responsibility

Reply:
Thanks @AshCorr I did it like this 
`def withNameOpt(s: String): Option[Value] = values.find(_.toString == s.toUpperCase)`

And now the caller doesn't need to do the `toUpperCase` any more